WordPress/src/main/java/org/wordpress/android/viewmodel/domains/DomainSuggestionsViewModel.kt

```diff
@@ -57,6 +60,11 @@ class DomainSuggestionsViewModel @Inject constructor(
 
     private var searchQuery: String by Delegates.observable("") { _, oldValue, newValue ->
         if (newValue != oldValue) {
+            if (isStarted && !isQueryTrackingCompleted) {
+                isQueryTrackingCompleted = true
+                AnalyticsTracker.track(Stat.DOMAIN_CREDIT_SUGGESTION_QUERIED)
```

Comment:
as mentioned above, I'd rather use `AnalyticsTrackerWrapper` because it gets injected